models/system/t5-small/description.md

```diff
@@ -20,9 +20,9 @@ Translation|Translation|[WMT16](https://huggingface.co/datasets/cnn_dailymail)|[
 
 ### Model Evaluation
 
-|Task|Use case|Dataset|Python sample (Notebook)|
-|---|--|--|--|
-|Translation|Translation|[wmt19/cs-en](https://huggingface.co/datasets/wmt19/viewer/cs-en/)|[evaluate-model-translation.ipynb](https://aka.ms/azureml-eval-sdk-translation)|
+| Task        | Use case    | Dataset                                                            | Python sample (Notebook)                                                        | CLI with YAML                                                                   |
+|-------------|-------------|--------------------------------------------------------------------|---------------------------------------------------------------------------------|---------------------------------------------------------------------------------|
+| Translation | Translation | [wmt19/cs-en](https://huggingface.co/datasets/wmt19/viewer/cs-en/) | [evaluate-model-translation.ipynb](https://aka.ms/azureml-eval-sdk-translation) | [evaluate-model-translation.ipynb](https://aka.ms/azureml-eval-sdk-translation) |[evaluate-model-translation.yml](https://aka.ms/azureml-eval-cli-translation)|
```

Comment:
Please update the CLI link at the end. Its pointing to notebook currently

Reply:
Which one? Checked the links and they are pointing to ymls.

Reply:
![image](https://user-images.githubusercontent.com/61145377/231399405-ff56a036-a749-48af-ab93-4228792fb9dd.png)